exporter/datadogexporter/utils/trace_helpers.go

```diff
@@ -107,3 +116,42 @@ func NormalizeSpanName(tag string) string {
 func NormalizeSpanKind(kind pdata.SpanKind) string {
 	return strings.TrimPrefix(kind.String(), "SPAN_KIND_")
 }
+
+// ported from https://github.com/DataDog/datadog-agent/blob/eab0dde41fe3a069a65c33d82a81b1ef1cf6b3bc/pkg/trace/traceutil/normalize.go#L72
+// fallbackServiceNames is a cache of default service names to use
+// when the span's service is unset or invalid.
+var fallbackServiceNames sync.Map
+
+// fallbackService returns the fallback service name for a service
+// belonging to language lang.
+func fallbackService(lang string) string {
```

Comment:
We should add tests for tis and `NormalizeSpanName` too. We can have unit tests for this by overriding the `fallbackServiceNames` variable or by having a `Normalizer` struct with the map as a field and `fallbackService` as a method.

Reply:
added some more tests for NormalizeSpanName